packages/desktop/src/main/invitation.ts

```diff
@@ -16,9 +16,11 @@ export const processInvitationCode = (mainWindow: BrowserWindow, code: string |
 
```

Comment:
This had nothing to do with the upgrade but I got tired of the errors so I cleaned this up.